### ISSUE TITLE
no-element-event-actions doesn't catch non action event handlers

### ISSUE THREAD

**Issue by mehulkar** [ember-template-lint/ember-template-lint#1343]
```html
<div onclick={{this.foo}}></div>
```

This snippet doesn't catch the `no-element-event-actions` rule, and I think it should be?

I think it's probably this line: https://github.com/ember-template-lint/ember-template-lint/blob/v2.7.0/lib/rules/no-element-event-actions.js#L38

**Comment by rwjblue** [comment]
Well, it isn't using `action` so that rule doesn't flag it 🤔. I guess in this case you really want `no-element-event-properties` kinda rule to completely forbid the `on***` style. 

tldr; This isn't a bug per-se (the rule as written wasn't supposed to catch this), but it is absolutely a gap in how we want to guide forwards with their templates.
